eth/ethconfig/gen_config.go

```diff
@@ -60,6 +60,9 @@ func (c Config) MarshalTOML() (interface{}, error) {
 		Checkpoint              *params.TrustedCheckpoint      `toml:",omitempty"`
 		CheckpointOracle        *params.CheckpointOracleConfig `toml:",omitempty"`
 		OverrideArrowGlacier    *big.Int                       `toml:",omitempty"`
+		MPTWitness              int
```

Comment:
What are `MPTWitness` and `CheckCircuitCapacity` used for here?

Reply:
yes